<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chessForm</class>
 <widget class="QWidget" name="chessForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="cbox_item">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>211</x>
        <y>48</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_pvp">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string>人人对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_pvc">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string>人机对战</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_pvn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string>网络对战</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
